### PR TITLE
FHAC-547:xml-api and xercesImpl jars are modified to provided scope to use jdk provided jars during runtime

### PR DESCRIPTION
**Pull request by alameluchidambaram**
1. xml-api and xercesImpl jars modified from compile, runtime to scope provided.
2. Tested on weblogic 12.1.3,websphere 8.5.5.3, wildfly 8.2.1-Final on Windows

### CODE REVIEW

File: Product/pom.xml [hunk 1]
```diff
@@ -292,6 +292,7 @@
                 <groupId>xerces</groupId>
                 <artifactId>xercesImpl</artifactId>
                 <version>2.10.0</version>
+                <scope>provided</scope>
```
Comment (christophermay07) [inline Product/pom.xml]: Scope of "provided" is not the right way to do this, we should exclude the jars where they are transitive dependencies.

**Comment by jsmith2** [comment]
Looks good to me.  As long as this has been confirmed on all app servers, I approve with no comments.

**Comment by christophermay07** [comment]
`org.connectopensource:CONNECTAdminGUI:war:4.6.0-SNAPSHOT`
`\- me.fhir:fhir-0.12:jar:0.3:compile`
`\- net.sf.saxon:Saxon-HE:jar:9.4:compile`
`\- xom:xom:jar:1.2.5:compile`
`\- xerces:xercesImpl:jar:2.8.0:compile`
Is this going to cause a problem if AdminGUI is deployed alongside CONNECT on WLS?

**Comment by sovanncgi** [comment]
This has been verified on all app servers.
https://cgiinterop.atlassian.net/browse/FHAC-587 has been created to address potential AdminGUI issues

**Pull request merged**